UserOperation.Web/wwwroot/js/site.js

```diff
@@ -1,5 +1,18 @@
 ﻿$(document).ready(function () {
     $("table").DataTable();
+<<<<<<< HEAD
```

Comment:
this is content generated by the merge conflicts. please remove it and fix the file